src/components/ebay-infotip/index.js

```diff
@@ -1,13 +1,12 @@
-const emitAndFire = require('../../common/emit-and-fire');
 
 module.exports = require('marko-widgets').defineComponent({
```

Comment:
Minor nit: there's whitespace at the beginning of this file.

Reply:
Our linter should catch this 😞